md/templates/md/agency_detail.html

```diff
@@ -110,6 +110,49 @@ <h4 class="graph-label">
   </div>
 {% endblock stop-display %}
 
+{% block stop-count-display %}
+  <div class="row">
+    <div class="col-md-12">
+    <h3>{% if officer_id %}Officer{% else %}Departmental{% endif %} Stop Count</h3>
+
+    <p class="help-block">
+      This graph displays the count of stop reasons by ethnic groups. Adjusting
```

Comment:
We are counting stops, not stop reasons, so it is a count of stops for each reason, not a count of stop reasons.